snippets/breadcrumbs.liquid

```diff
@@ -0,0 +1,17 @@
+<div class="breadcrumbs-container">
+  <div class="breadcrumbs-listings">
+    <div class="listings__item">
+      <a href="/" class="first-listing">الرئيسية</a>
+    </div>
+    <div class="listings__item">
+      <img src='{{ 'left-arrow.png' | asset_url }}'>
+    </div>
+    <div class="listings__item">
+      <a href="/collections" class="second-listing">التصنيفات</a>
```

Comment:
this should not be static it should be getting the category name

Reply:
we are not inside a category we are showing all the collections

Reply:
ok but should be translated too